mtags/src/main/scala-3/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -15,6 +16,7 @@ import scala.meta.pc.*
 
 import dotty.tools.dotc.ast.tpd.*
 import dotty.tools.dotc.ast.untpd
+import dotty.tools.dotc.core.Constants
```

Comment:
Are `scala.collection.immutable.LazyList.cons` and `dotty.tools.dotc.core.Constants` needed?